spec/falcon/adapters/input_spec.rb

```diff
@@ -102,6 +112,14 @@
 				
 				expect(subject).to be_eof
 			end
+			
+			it "allocates little memory" do
+				sample_data.replace Array.new(5) { |i| "#{i}" * 1024*1024 }
```

Comment:
Rather than doing this, put it in a different context and `let(:sample_data) {Array.new ....}`.

Ideally, don't nest the context too much because it makes the output a bit clunky.